src/main/java/com/google/api/gax/grpc/BundlingSettings.java

```diff
@@ -103,6 +103,12 @@
   /** Returns the Boolean object to indicate if the bundling is enabled. Default to true */
   public abstract Boolean getIsEnabled();
 
+  public abstract Boolean getIsFlowControlEnabled();
```

Comment:
@pongad I think we should have an 'enabled' flag for flow control like we do for bundling. There are two reasons:

1. It is easier to switch between enabled/disabled if something needs to be investigated or whatever - it's not necessary to muck around with the settings themselves. If you set all the values to null, then later when you want to turn it on again, you have to go find the original values.
2. If a new setting is created after the user has written the code to set existing settings to null, the new setting won't be nulled out and could take effect.

That said, @michaelbausor , I think the enabled flag should live on the `FlowControlSettings` class itself. Otherwise, the flag would have to be repeated in every place that contained flow control settings.